stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -0,0 +1,1056 @@
+    }
+}
+
+pub const NAKAMOTO_CHAINSTATE_SCHEMA_1: &'static [&'static str] = &[
+    r#"
+      -- Table for Nakamoto Block Headers
+      CREATE TABLE nakamoto_block_headers (
+          -- The following fields all correspond to entries in the StacksHeaderInfo struct
+                     block_height INTEGER NOT NULL,
+                     -- root hash of the internal, not-consensus-critical MARF that allows us to track chainstate/fork metadata
+                     index_root TEXT NOT NULL,
+                     -- all consensus hashes are guaranteed to be unique
+                     consensus_hash TEXT UNIQUE NOT NULL,
+                     -- burn header hash corresponding to the consensus hash (NOT guaranteed to be unique, since we can 
+                     --    have 2+ blocks per burn block if there's a PoX fork)
+                     burn_header_hash TEXT NOT NULL,
+                     -- height of the burnchain block header that generated this consensus hash
+                     burn_header_height INT NOT NULL,
+                     -- timestamp from burnchain block header that generated this consensus hash
+                     burn_header_timestamp INT NOT NULL,
+                     block_size TEXT NOT NULL,
+          -- The following fields all correspond to entries in the NakamotoBlockHeader struct
+                     version INTEGER NOT NULL,
+                     -- this field is the total number of blocks in the chain history (including this block)
+                     chain_length INTEGER NOT NULL,
+                     -- this field is the total amount of BTC spent in the chain history (including this block)
+                     btc_spent INTEGER NOT NULL,
+                     -- the parent BlockHeaderHash
+                     parent TEXT NOT NULL,
+                     -- the latest bitcoin block whose data is viewable from this stacks block
+                     burn_view TEXT NOT NULL,
+                     -- stackers' signature over the block
+                     signature TEXT NOT NULL,
+                     tx_merkle_root TEXT NOT NULL,
+                     state_index_root TEXT NOT NULL,
+
+          -- The following fields are not part of either the StacksHeaderInfo struct
+          --   or its contained NakamotoBlockHeader struct, but are used for querying
+                     header_type TEXT NOT NULL,
+                     block_hash TEXT NOT NULL,
+                     -- index_block_hash is the hash of the block hash and consensus hash of the burn block that selected it, 
+                     -- and is guaranteed to be globally unique (across all Stacks forks and across all PoX forks).
+                     -- index_block_hash is the block hash fed into the MARF index.
+                     index_block_hash TEXT NOT NULL,
+                     -- the total cost of the block
+                     cost TEXT NOT NULL,
```

Comment:
Can you explain more what `cost` means here?

Reply:
Ah, I see -- this is the serialized `ExecutionCost` that represents how much runtime this block consumed, right?

Reply:
Yep, exactly -- this is the same as the current `cost` field in the existing headers table.